mne/decoding/csp.py

```diff
@@ -171,7 +171,7 @@ def fit(self, X, y):
                 weight = len(class_)
 
             # normalize by trace and stack
```

Comment:
This comment probably needs to change, too :)

Maybe just mention that trace normalization can break some use cases so we omit it here